include/exadg/postprocessor/pointwise_output_generator_base.cpp

```diff
@@ -145,8 +145,10 @@ PointwiseOutputGeneratorBase<dim, Number>::setup_base(
 #else
   (void)triangulation_in;
   (void)mapping_in;
-  (void)pointwise_output_data_in;
-  AssertThrow(false, dealii::ExcMessage("deal.II is not compiled with HDF5!"));
+  if(pointwise_output_data_in.time_control_data.is_active)
```

Comment:
Is the code in lines 102-114 actually placed correctly? Why not inside the if-statement (line 116)?

I think the `#ifdef` should also appear inside the if-statement above since this would make the coder easier to understand for the `#else` branch.

Reply:
Yep; I think you are right. I think we should move 102-114 and the `#ifdef' inside the if.